css/main.css

```diff
@@ -1,3 +1,263 @@
-/* Add your CSS here */
+*:root {
+  --main-bg-color: #fff;
+  --primary: #e6dacd; /*dark beige*/
+  --secondary: #f4ece6; /*light beige*/
+  --accent-colour: #0150fd; /*royal beige*/
+  --body-color: #000;
+}
 
-/* Dont' forget to link this file to your HTML in the <head> */
+body {
+  font-family: "Quicksand", sans-serif;
+  margin: 0;
+  padding: 0;
+}
+
+.square-h1 {
+  display: flex;
+  flex-direction: row;
+  gap: 5px;
+}
+
+.homepage-main {
+  display: grid;
+  grid-template-columns: repeat(2, 1fr);
+  grid-template-rows: 126.4px 952px;
+
+  grid-template-areas:
+    "header header "
+    "content  aside"
+    "footer footer";
+}
+
+.homepage-header {
+  background-color: var(--main-bg-color);
+  grid-column-end: -1;
+  grid-column-start: 1;
+  grid-row: 1;
+}
+
+.homepage-content {
+  background-color: var(--secondary);
+  margin-left: 10px;
+  grid-column-start: 1;
+  grid-column-end: -1;
+  padding: 10px;
+  width: 100vw;
+  margin: 20px;
+  width: 280px;
+  height: 375px;
+  grid-row: 2;
+}
+
+.homepage-aside {
+  background-color: var(--primary);
+  grid-column-start: 1;
+  grid-column-end: -1;
+  margin: -500px 0 0 0;
+  padding: 10px;
+}
+
+.homepage-footer {
+  grid-column-start: 1;
+  grid-column-end: -1;
+  background-color: var(--main-bg-color);
+}
+
+h1 {
+  font-family: "Poppins", sans-serif;
+}
+
+.contact {
+  display: flex;
+}
+
+.social-footer {
+  width: 20px;
+  height: 20px;
+}
+
+a {
+  text-decoration: none;
+}
+
+li {
+  display: inline-flex;
+  flex-flow: row;
+  flex-wrap: nowrap;
+  list-style: none;
+  padding: 5px;
+  margin: 5px;
+}
+
+.ul {
+  width: 451px;
+  height: 35px;
+}
+
+hr {
+  color: var(--accent-colour);
+  size: 10px;
+}
+
+.font-0 {
+  font-size: 40px;
+  font-family: "Poppins";
+}
+
+.navbar {
+  display: none;
+}
+
+.blue-square {
+  background-color: var(--accent-colour);
+  height: 16px;
+  width: 16px;
+  margin-top: 30px;
+}
+
+.img-hero {
+  width: 120px;
```

Comment:
adding width and height directly into an image is not recommended, as I explained previously